tests/checkpoint/test_checkpoint.py

```diff
@@ -1450,9 +1460,300 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_with_
     assert validation_result.meta["checkpoint_name"] == checkpoint_name
 
 
+@pytest.mark.slow  # 1.15s
+def test_newstyle_checkpoint_raises_error_if_batch_request_and_validator_are_specified_in_constructor(
+    titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled,
+    batch_request_as_dict,
+):
+    context: DataContext = titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled
```

Comment:
DataContext is not a type anymore - this is a FileDataContext

Reply:
@cdkini I changed this universally in this module and in other checkpoint test modules -- thanks.